collectors/build/jenkins-audit/pom.xml

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>Hygieia</artifactId>
+        <groupId>com.capitalone.dashboard</groupId>
+        <version>2.0.5-SNAPSHOT</version>
+        <relativePath>../../../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <name>${project.groupId}:${project.artifactId}</name>
+    <description>Jenkins Audit Collector</description>
+    <url>https://github.com/capitalone/Hygieia</url>
+    <packaging>jar</packaging>
+
+
+    <artifactId>jenkins-audit-collector</artifactId>
+
+    <distributionManagement>
+        <snapshotRepository>
+            <id>ossrh</id>
+            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
```

Comment:
please check if these repo's are required in this project

Reply:
removed these repos